includes/polling/applications/sagan.inc.php

```diff
@@ -88,13 +90,14 @@
 
 // if we have any instance changes, log it
 if (sizeof($added_instances) > 0 or sizeof($removed_instances) > 0) {
-    $app->data = ['instances' => $instances];
     $log_message = 'Sagan Instance Change:';
     $log_message .= count($added_instances) > 0 ? ' Added ' . json_encode($added_instances) : '';
     $log_message .= count($removed_instances) > 0 ? ' Removed ' . json_encode($added_instances) : '';
     log_event($log_message, $device, 'application');
 }
 
+$app->data = $new_data;
```

Comment:
looks like the $new_data variable doesn't do anything, so you can just inline it to make it easier to follow.

Reply:
@murrant Done.